addons/purchase/migrations/12.0.1.2/pre-migration.py

```diff
@@ -53,8 +53,23 @@ def precompute_pol_product_uom_qty(env):
             AND pol.product_uom != pt.uom_id""",
     )
 
```

Comment:
add blank line to solve flake8

Reply:
I cannot believe the problem was that minor thing. :sob: